spec/controllers/admin/log_entries_controller_spec.rb

```diff
@@ -26,6 +26,55 @@ def post_create(action: "create", logeable: create(:customer))
         post_create
         expect(response).to redirect_to([:admin, LogEntry.last])
       end
+
+      it "parses nested polymorphic resources" do
+        customer = create(:customer)
+
+        resource_params = attributes_for(:log_entry).merge(
+          arbitrarily: {
+            nested: {
+              params: {
+                logeable: {
+                  type: "Administrate::Field::Polymorphic",
+                  value: customer.to_global_id.to_s,
+                },
+              }
+            }
+          }
+        )
+
+        allow_any_instance_of(
+          LogEntryDashboard
+        ).to receive(:permitted_attributes).and_return(
+          [
+            arbitrarily: {
+              nested: {
+                params: {
+                  logeable: [
+                    :type,
+                    :value,
+                  ]
+                }
+              }
+            }
+          ]
+        )
+
+        begin
+          post :create, log_entry: resource_params
+        rescue ActiveModel::UnknownAttributeError
+          # Ignore errors that arise because the parameters are contrived to
+          # test that nested polymorphic parameters are properly converted
```

Comment:
Does this happen with this test? I wonder if there's a way around it if so

Reply:
This does happen, because the params that are being used to attempt to create a LogEntry record are fictitious and don't align with viable attributes. I'll try tossing spinning up an `attr_accessor` on the model, my first thought was to mock in the entire NestedHasMany logic to make those params valid but now that you ask I think there's probably a quick workaround.

Reply:
@nickcharlton `attr_accessor` worked great, pushed a commit up for that + hound fixes. Only hound stuff left looks ignorable (I think).